frontend/src/components/world/WorldMap.tsx

```diff
@@ -379,6 +395,17 @@ class CoveyGameScene extends Phaser.Scene {
     // Watch the player and worldLayer for collisions, for the duration of the scene:
     this.physics.add.collider(sprite, worldLayer);
 
+    /* Configure physics overlap behavior for when the player steps into
```

Comment:
Not necessarily needed in this PR but how will we indicate to the user that this is what they should do?

Reply:
i can change the textbox with instructions to display "Press space near the blue sign to open leaderboard"